plugin/incsearch.vim

```diff
@@ -71,6 +71,8 @@ noremap <Plug>(_incsearch-#)  #
 noremap <Plug>(_incsearch-g*) g*
 noremap <Plug>(_incsearch-g#) g#
 
+noremap <silent> <Plug>(_incsearch-#-searchforward) :<C-u>let v:searchforward=g:incsearch#consistent_n_direction<CR>
```

Comment:
I'm really sorrrrry for the late reply :sweat_drops: 
Can you fix this line to support `c`(c#) operator and visual mode(v#)?

```vim
nnoremap <silent> <Plug>(_incsearch-#-searchforward) :<C-u>let v:searchforward=g:incsearch#consistent_n_direction<CR>
xnoremap <silent> <Plug>(_incsearch-#-searchforward) :<C-u>let v:searchforward=g:incsearch#consistent_n_direction<CR>gv
inoremap <silent> <Plug>(_incsearch-#-searchforward) <Nop>
```